libpod/container_api.go

```diff
@@ -144,6 +144,13 @@ func (c *Container) Init() (err error) {
 	}
 	g.AddMount(hostnameMnt)
 
+	// Bind builtin image volumes
+	if c.config.ImageVolumes {
```

Comment:
Is this a list? Shouldn't you be checking if len(c.config.ImageVolumes) > 0

Reply:
No this is a boolean, its `ctr.config.ImageVolumes`, you are thinking about `createConfig.ImageVolumes` which is a list. I will rename the variable as it is confusing.

Reply:
fixed